.github/ISSUE_TEMPLATE/report_a_bug.yml

```diff
@@ -35,68 +25,10 @@ body:
     id: description
     attributes:
       label: Description
-      description: Provide a clear and concise description of the issue.
-    validations:
-      required: true
-
-  - type: textarea
-    id: expectation
-    attributes:
-      label: Expectation
-      description: Tell us about the behavior you expected to see.
+      description: Describe how to reproduce your issue, what you are seeing and what you expected to see. Please do not report security vulnerabilities here; use https://github.com/openfga/openfga/security/advisories/new or send us an email at security@openfga.dev instead
```

Comment:
Do we want to re-emphasize do not send unexpected check / list objects / list users result here (in case someone skip over the description)?